src/sunstone/public/app/utils/quotas/quota-widgets.js

```diff
@@ -506,6 +506,8 @@ define(function(require) {
           emptyValue: true
         });
 
+      $(".ds_select .resource_list_select option[elem_id='0']", context).remove();
```

Comment:
does this cover all system DSs, or just DS with ID = 0?

Reply:
Just '0: system' DS